python/src/aiconfig/editor/server/server.py

```diff
@@ -1,5 +1,6 @@
 import logging
 from typing import Any, Dict, Type, Union
+import webbrowser
```

Comment:
add to requirements for editor

Reply:
standard lib

https://docs.python.org/3/library/webbrowser.html